docker-compose.tmpl.yml

```diff
@@ -57,14 +60,23 @@ services:
       <<: *logging-env
       hive_service: cron
 
-  img_upload_app:
-    image: beabee/pictshare:v0.3.0
+  # MinIO service
+  minio:
+    image: beabee/beabee-minio:${HIVE_VERSION:-__VERSION__}
     restart: unless-stopped
-    environment:
-      URL: ${BEABEE_AUDIENCE}/uploads/
-      CONTENTCONTROLLERS: ${BEABEE_UPLOAD_CONTENTCONTROLLERS:-IMAGE}
     volumes:
-      - upload_data:/var/www/data
+      - minio_data:/data
+    healthcheck:
```

Comment:
Just for completeness: healthcheck can be moved into build step (I think), as per https://github.com/beabee-communityrm/monorepo/pull/163